extensions/ql-vscode/test/pure-tests/databases/data/workspace-databases.json

```diff
@@ -8,5 +8,39 @@
     ],
     "owners": [],
     "repositories": ["owner/repo1", "owner/repo2", "owner/repo3"]
+  },
+  "local": {
+    "lists": [
+      {
+        "name": "localList1",
+        "databases": [
+          {
+            "name": "foo/bar",
+            "dateAdded": 1668096745193,
+            "language": "go",
+            "path": "Uri to db in workspace storage"
```

Comment:
I guess it could just be the file system path rather than a URI? What do we store currently?

Reply:
We currently store a `Uri`: https://github.com/github/vscode-codeql/blob/7e8fa5c5fe39f89273d817ae09b74aac0bb4a5fb/extensions/ql-vscode/src/databases.ts#L206

We could change to a file system path, but I wonder if that would make it harder to integrate with existing stuff?

Reply:
Hmm `Uri` is a VS Code thing so I'm thinking we should stick to filesystem paths. We can easily translate to `vscode.Uri` by doing `vscode.Uri.file('/path/to/dir')`.

Reply:
Ah, good point. Will change that 👍🏽